go.mod

```diff
@@ -53,11 +52,11 @@ require (
 	github.com/openshift/client-go v0.0.0-20201020074620-f8fd44879f7c
 	github.com/openshift/cloud-credential-operator v0.0.0-20200316201045-d10080b52c9e
 	github.com/openshift/cluster-api v0.0.0-20191129101638-b09907ac6668
-	github.com/openshift/cluster-api-provider-gcp v0.0.1-0.20200120152131-1b09fd9e7156
+	github.com/openshift/cluster-api-provider-gcp v0.0.1-0.20201002065957-9854f7420570
 	github.com/openshift/cluster-api-provider-libvirt v0.2.1-0.20191219173431-2336783d4603
 	github.com/openshift/cluster-api-provider-ovirt v0.1.1-0.20200504092944-27473ea1ae43
 	github.com/openshift/library-go v0.0.0-20201022113156-a4ff9e1d2900
-	github.com/openshift/machine-api-operator v0.2.1-0.20200429102619-d36974451290
+	github.com/openshift/machine-api-operator v0.2.1-0.20201002104344-6abfb5440597
```

Comment:
What made you choose this particular commit?

Reply:
https://github.com/openshift/cluster-api-provider-aws/blob/master/go.mod#L13

Reply:
I picked this one to comply with cluster-api-provider-aws

Reply:
I would have thought the hierarchy would have been reversed. I would have thought that you would pick the latest machine-api-operator and then select the cluster-api-providers to match what that machine-api-operator version is using. I guess I don't care much either way, since both solve the issue of removing the deleted goautoneg module.

Reply:
done! machine api operator is on the latest commit now